.github/workflows/ci.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
         runs-on: ubuntu-latest
 
         strategy:
+            fail-fast: false
```

Comment:
Are the changes in this file necessary?

Reply:
bzlmod was failing on CI; we should add fast-fail throughout the repos actually. I'll prefactor it.